Tests/Pcap++Test/Tests/IpMacTests.cpp

```diff
@@ -40,12 +40,21 @@ PTF_TEST_CASE(TestIPAddress)
 	PTF_ASSERT_EQUAL(ip4AddrFromIpAddr, secondIPv4Address);
 
 	pcpp::IPv4Address ipv4Addr("10.0.0.4"), subnet1("10.0.0.0"), subnet2("10.10.0.0"), mask("255.255.255.0");
+	std::string maskedSubnet1("10.0.0.0/24"), maskedSubnet2("10.10.0.0/24");
+	std::string invalidMaskedSubnet1("aaaa"), invalidMaskedSubnet2("10.0.0.0"), invalidMaskedSubnet3("10.0.0.0/aa"), invalidMaskedSubnet4("10.0.0.0/33"), invalidMaskedSubnet5("999.999.1.1/24");
 	PTF_ASSERT_TRUE(ipv4Addr.isValid());
 	PTF_ASSERT_TRUE(subnet1.isValid());
 	PTF_ASSERT_TRUE(subnet2.isValid());
 	PTF_ASSERT_TRUE(mask.isValid());
 	PTF_ASSERT_TRUE(ipv4Addr.matchSubnet(subnet1, mask));
 	PTF_ASSERT_FALSE(ipv4Addr.matchSubnet(subnet2, mask));
+	PTF_ASSERT_TRUE(ipv4Addr.matchSubnet(maskedSubnet1));
+	PTF_ASSERT_FALSE(ipv4Addr.matchSubnet(maskedSubnet2));
+	PTF_ASSERT_FALSE(ipv4Addr.matchSubnet(invalidMaskedSubnet1));
+	PTF_ASSERT_FALSE(ipv4Addr.matchSubnet(invalidMaskedSubnet2));
+	PTF_ASSERT_FALSE(ipv4Addr.matchSubnet(invalidMaskedSubnet3));
+	PTF_ASSERT_FALSE(ipv4Addr.matchSubnet(invalidMaskedSubnet4));
+	PTF_ASSERT_FALSE(ipv4Addr.matchSubnet(invalidMaskedSubnet5));
```

Comment:
Please add `pcpp::Logger::getInstance().suppressLogs()` before the negative tests and `pcpp::Logger::getInstance().enableLogs()`, same as we do here for example: https://github.com/seladb/PcapPlusPlus/blob/b1c4e99c40071bf0639b773c17837f096d4f8cfa/Tests/Pcap%2B%2BTest/Tests/RawSocketTests.cpp#L27-L32

Otherwise the test output contains log errors:

![image](https://user-images.githubusercontent.com/9059541/207802139-ea3ebfbe-a8ff-484a-88c0-ff8954d35290.png)

https://github.com/seladb/PcapPlusPlus/actions/runs/3694904138/jobs/6271844098